### ISSUE TITLE
GooglePresent template doesn't work - the URL structure from Google has changed

### ISSUE THREAD

**Issue by hendrikknoche** [SteffenLehmann/Generico-Filter-Templates#8]
Before the URLs looked like this:
https://docs.google.com/presentation/d/e/2PACX-1vTyEwM3LgfTalzzLuDvl3UBx_9S09q6Ca_8ZL4GWR-T7TC3AKuMMqS7YKJL5HnAZkEErZGOhhDSsvT7/pub?start=false&loop=false&delayms=3000

now they are like this
https://docs.google.com/presentation/d/1vEby7pTWJroJiKgBsckit0P7IUJJHnBRyoVfuhoGbSo/pub?start=false&loop=false&delayms=3000

instead of the /d/e/2PACX- it only has /d/... and a much shorter ID.

**Comment by hendrikknoche** [comment]
addressed in https://github.com/SteffenLehmann/Generico-Filter-Templates/commit/a8364deda14a61275b23dfb3638f36b6d90149da

**Comment by SteffenLehmann** [comment]
Great, I will close the issue then. :)

**Comment by hendrikknoche** [comment]
Currently the code can only handle the new format of URLs from the Google suite - in case you published to web before ca.  summer of 2024 then the old URL structure which includes /d/e/ will not be recognized by this template. This would require more work than can handle both formats

**Comment by SteffenLehmann** [comment]
I see, thanks for clarifying. I will update it accordingly.

**Comment by hendrikknoche** [comment]
I have updated the CustomJS part for this and it's working. But I haven't updated the matching bundle. I don't understand it's syntax.

**Comment by SteffenLehmann** [comment]
Just to be clear, you want both the legacy and 2023 version of the googlepresent - Google Publish to web update?

https://github.com/SteffenLehmann/Generico-Filter-Templates/blob/main/Legacy%20templates/GooglePublishToWeb.txt 
https://github.com/SteffenLehmann/Generico-Filter-Templates/tree/main/Templates/GooglePublishToWeb

**Comment by hendrikknoche** [comment]
yes, please. maybe this is already working in the 2023 version but please check.
